fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -3046,7 +3046,7 @@ nonReserved
     | JOB
     | LABEL | LAST | LESS | LEVEL | LIST | LOCAL | LOCATION | LOGS | LOGICAL | LOW_PRIORITY | LOCK | LOCATIONS
     | MANUAL | MAP | MAPPING | MAPPINGS | MASKING | MATCH | MAPPINGS | MATERIALIZED | MAX | META | MIN | MINUTE | MINUTES | MODE | MODIFY | MONTH | MERGE | MINUS | MULTI_COLUMNS
-    | NAME | NAMES | NEGATIVE | NO | NODE | NODES | NONE | NULLS | NUMBER | NUMERIC
+    | NAME | NAMES | NEGATIVE | NO | NODE | NODES | NONE | NULLS | NUMBER | NUMERIC | NO_VALIDATION
```

Comment:
how about `RETAIN_VERSION_MAP` to be more clear?

Reply:
https://sider.ai/share/da2fb07888f04fa159526834f5ee6475

I consulted Deepseek and received the following answers; perhaps we can choose one of them.
![image](https://github.com/user-attachments/assets/52ed82d7-f071-4abe-85f0-f941516ea1bf)